PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.c

```diff
@@ -211,6 +211,7 @@ PcRtcInit (
   UINT32          TimerVar;
   BOOLEAN         Enabled;
   BOOLEAN         Pending;
+  BOOLEAN         NeedRtcUpdate = FALSE;

```

Comment:
Does not follow coding style to pre-init local.  Please move setting to FALSE to beginning of function.